heliopy/util/config.py

```diff
@@ -51,14 +52,12 @@ def load_config():
     config_dict = {}
 
     # Set data download directory
-    download_dir = os.path.expanduser(config['DEFAULT']['download_dir'])
-    if os.name == 'nt':
-        download_dir = download_dir.replace('/', '\\')
+    download_dir = pathlib.Path(config['DEFAULT']['download_dir']).expanduser()
     config_dict['download_dir'] = download_dir
```

Comment:
Again, to fix stuff elsewhere could just convert `download_dir` to a string here.